AdobeStockImageAdminUi/view/adminhtml/web/template/grid/column/image-preview.html

```diff
@@ -58,23 +58,44 @@ <h3 class="title" text="$col.getTitle($row())" if="$col.getTitle($row())"/>
         <div class="keywords">
             <div class="title" translate="'Similar Keywords'"></div>
             <!-- ko foreach: $col.getKeywords($row()) -->
-                <div class="keyword" data-bind="css: { 'hide': $index() >= $col.getKeywordsLimit($row())}">
-                    <a href="" class="value" data-bind="click: function(){ $col.searchByKeyWord(name) }">
-                        <span text="name"/>
-                    </a>
-                </div>
+            <div class="keyword" data-bind="css: { 'hide': $index() >= $col.getKeywordsLimit($row())}">
+                <a href="" class="value" data-bind="click: function(){ $col.searchByKeyWord(name) }">
+                    <span text="name"/>
+                </a>
+            </div>
             <!-- /ko -->
             <button data-bind="visible: $col.canViewMoreKeywords($row()), click: function(){ $col.viewAllKeywords($row()) }">
                 <span translate="'View all'"/>
             </button>
         </div>
-        <div class="series">
-            <div class="title" translate="'More from this series'"></div>
-            <!-- ko foreach: $col.getSeries($row()) -->
-            <div class="thumbnail" data-bind="css: { 'hide': $index() >= 3}">
-                <img attr="src: thumbnail_url, alt: title">
+        <div id="adobe-stock-tabs" attr="'data-mage-init': getDataMageInit()">
```

Comment:
That is incorrect usage of `data-mage-init` attribute. I think `data-mage-init` attribute should be static in this case.